src/crush/CrushWrapper.cc

```diff
@@ -1143,17 +1143,17 @@ int CrushWrapper::insert_item(
 
     if (!name_exists(q->second)) {
       ldout(cct, 5) << "insert_item creating bucket " << q->second << dendl;
-      int cur_weight = 0, newid;
+      int cur_weight = 0, bucketid;
```

Comment:
```suggestion
      int cur_weight = 0, new_bucket_id;
```

but i like `newid` also. as it emphasizes that its an id for the newly added bucket. and its lexical scope is very small, while "bucketid" loses the "new" in it, the new name does not help the reader understand that this bucket is a newly allocated one. so "newid" should be fine as well.

Reply:
@tchaikov 
I'll use "new_bucket_id" here.
The function name here is ```CrushWrapper::insert_item```, item includes devices and buckets in crushmap. My purpose is to hint that this is bucket instead of device.